### DISCUSSION TITLE
[Bug] Namespace in model fields

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#389] [Help] [answered]
At editing a model (in my case `com.products`) custom field, the API namespace under "Advanced options", set at creation is not selected anymore. So if one does not pay attention the namespace changes from i.e "Content" to "None" at saving.

**Comment by logeshswell** [comment]
@josias-hosang Yes, it is a bug. I will file a bug ticket and will keep you posted if I get some updates.

**Comment by steven-k1** [comment, marked as answer]
This has been resolved
